### DISCUSSION TITLE
Set up my mastodon website. All I see is a login page. Logged in and it just takes me to an edit page

### DISCUSSION THREAD

**Discussion by jsailo** [mastodon/mastodon#19114] [Q&A]
So I have set up my mastodon site. On going to the site, it takes me to https://mysite.com/about.  If I login with my admin credentials, all I see is an Account settings page (see pic below). 

1. What do I need so users can start registering, posting, or _connect_(?) to other instances?
2. If I click on **Didn't receive confirmation instructions?**, I'm taken to https://mysite.com/auth/confirmation/new where I enter my email but no mail is sent.

**Account Settings page**
In the below screen, if I click **Back to Mastodon**, it just takes me to https://mysite.com/auth/edit
What else do I need to do here?

![image](https://user-images.githubusercontent.com/35369823/188325621-8da4fcbb-adf6-436b-8ad5-b98ea40b2258.png)

**Comment by jazmichaelking** [comment]
You need to respond to the email confirmation. If you didn't receive the email, click the link at the top in your screenshot.

**Reply by jsailo** [reply]
I clicked the email but nothing happened. Copy/pasting my comment from elsewhere (in case someone lands here).

===

It seems mastodon has updated its permission system and you now need to do this manually via `tootctl`.

I was told to run this:

`tootctl accounts modify owner_username --role Owner`. 

The above didn't work for me but the following worked. 

`RAILS_ENV=production bin/tootctl accounts modify owner_username --role Owner`

Run the above as `su - mastodon` in the `live` directory.

Where, **owner_username** is the username you want to set as the owner of the site.

You will now be able to access the `/admin/settings/edit` page of your site. 

However, the **ADMINISTERED BY** will still be blank or labelled as **Not set** on the about page of your site.

For this, update the **[Contact username](https://i.imgur.com/fvZ3uyi.png)** field in the `/admin/settings/edit` page.

**Reply by sndavis** [reply]
@jsailo where did you run the above? I am also stuck on the account details page without any confirmation instructions

**Reply by MrGeneration** [reply]
Note that in Mastodon `3.5.3` the role owner doesn't seem to exist:

`Expected '--role' to be one of user, moderator, admin; got owner`

**Comment by jsailo** [comment]
@sndavis if you mean the following command. 

`RAILS_ENV=production bin/tootctl accounts modify owner_username --role Owner`

I ran it in the `live` directory. Set the role to `admin` as I didn't see the Administration items when I set it to `owner`.
So, use the following: 

`RAILS_ENV=production bin/tootctl accounts modify owner_username --role admin`
